### PR TITLE
Added token_type parameter to OAuth access token request endpoint

### PR DESCRIPTION
**Pull request by clintonb**
This parameter will enable clients to request alternative token types (e.g. JWT).

ECOM-4221

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @efagin

File: swagger/oauth.yaml [hunk 1]
```diff
@@ -29,6 +29,10 @@ endpoints:
           type: string
           format: password
         - in: formData
+          name: token_type
+          required: false
+          type: string
+        - in: formData
```
Comment (efagin) [inline swagger/oauth.yaml]: It's in the post body? I thought it was in the query string?

Comment (clintonb) [inline swagger/oauth.yaml]: We are already sending a post body, so it made more send to extend it rather than add a GET parameter.

Comment (efagin) [inline swagger/oauth.yaml]: Got it. Cool.

**Pull request merged**